src/constructN.js

```diff
@@ -24,7 +24,7 @@ var nAry = require('./nAry');
  *      };
  *      Salad.prototype.recipe = function() {
  *        var instructions = R.map((ingredient) => (
- *          'Add a whollop of ' + ingredient, this.ingredients)
+ *          'Add a whollop of ' + ingredient), this.ingredients
```

Comment:
two questions: 
1. I wonder why this isn't all on one line?
2. What is a "whollop"? I've heard of "dollop" ...

I realize this is how it was already, but as long as we're in there fixing it ...